### PR TITLE
Renames R_VAREDIT to R_RCE

### PR DESCRIPTION
**Pull request by Cyberboss**
Sensible

### CODE REVIEW

File: code/__DEFINES/admin.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #define R_PERMISSIONS	0x80
 #define R_STEALTH		0x100
 #define R_POLL			0x200
-#define R_VAREDIT		0x400
+#define R_RCE		0x400
```
Comment (kevinz000) [inline code/__DEFINES/admin.dm]: Wtf admins are responsible!!!??

**Comment by MrStonedOne** [comment]
Context?

**Comment by Cyberboss** [comment]
Shitposting

**Comment by MrStonedOne** [comment]
i know that, i was wondering what the context for the shit post was

**Pull request closed without merging**